.github/workflows/main.yml

```diff
@@ -0,0 +1,63 @@
+name: Run tests
+
+on:
+  [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: Ubuntu-20.04
+
+    env:
+      DB_TYPE: postgresql
+      DB_USERNAME: postgres
+      DB_PASSWORD: postgres
+      DB_ENDPOINT: localhost:5432 
+      DB_NAME: bitschema
```

Comment:
here we only need DB_TEST_NAME. DB_NAME is not being used coz gh-action only run the tests.

Reply:
ahh okay will update it